gnovm/stdlibs/time/timezoneinfo.gno

```diff
@@ -63,6 +63,14 @@ const (
 	omega = 1<<63 - 1 // math.MaxInt64
 )
 
+// Local represents the chain's local time zone.
```

Comment:
Help Git by adding the line back at the same location as before. It should be a no-diff instead of a dual diff.